### PR TITLE
Added BITCAR token to default tokens

### PR DESCRIPTION
**Pull request by wfoster232**

### CODE REVIEW

File: dist/tokens/eth/tokens-eth.json [hunk 1]
```diff
@@ -2068,6 +2068,40 @@
       "youtube": "https://www.youtube.com/channel/UCFvNUWRiedqvcpNHqLnDAXg"
     }
   },
+  {
+    "symbol": "BITCAR",
+    "name": "BitCar Token",
```
Comment (gamalielhere) [inline dist/tokens/eth/tokens-eth.json]: Create the file for this in `src/tokens/eth/<tokenaddress>.json`

**Comment by wfoster232** [comment]
Hi @gamalielhere,

I have added that file as requested.

Thanks

**Comment by wfoster232** [comment]
Hi @gamalielhere 

Would it be possibly to get this committed before another merge conflict occurs?

Thanks

File: dist/tokens/eth/tokens-eth.json [hunk 2]
```diff
@@ -2822,6 +2822,40 @@
       "youtube": ""
     }
   },
+  {
+    "symbol": "BITCAR",
```
Comment (gamalielhere) [inline dist/tokens/eth/tokens-eth.json]: Remove this change

**Comment by gamalielhere** [comment]
@wfoster232 There should be no conflicts in this current set up. you shouldn't touch the `dist` folders since we compile it locally. I'll update the README soon to make it clear for everyone else too.

**Comment by wfoster232** [comment]
Resolved, thanks for clarifying.

**Pull request merged**